packages/core-contracts/contracts/satellite/SatelliteFactory.sol

```diff
@@ -1,6 +1,16 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+struct Satellite {
+    bytes32 id;
+    string contractName;
+    address deployedAddress;
+}
+
 abstract contract SatelliteFactory {
```

Comment:
IMO keep SatelliteFactory only, and always return an array

Reply:
Done!